### DISCUSSION TITLE
Call a plugin from an extension

### DISCUSSION THREAD

**Discussion by mikael-desharnais** [eclipse-theia/theia#8126] [General] [answered]
Hi,

Is there a way to call a plugin API from an extension ? Following the documentation, I was able to do this between two plugins but using HostedPluginDeployerHandler I can get a lot of information concerning the plugin but not its exported API.

Thanks

**Comment by akosyakov** [comment, marked as answer]
You cannot access the exported API, but your plugin can register commands which can be called by an extension then and vice versa.
